lib/repositories/match_details_repository.dart

```diff
@@ -165,4 +166,162 @@ class MatchDetailsRepository {
     ];
     return response.map(MatchEventsModel.fromJson).toList();
   }
+
+  Future<List<LineUpModel>> fetchMatchLineUps(String matchID) async {
+    // return (await liveMatchesRemoteService.fetchMatchLineUps(matchID: matchID)).response;
+    final response = [
```

Comment:
You're going to eventually call some API for this right? Is it a mock to test your UI?

Reply:
exactly